google_cloud_compute/komand_google_cloud_compute/util/api.py

```diff
@@ -0,0 +1,195 @@
+from googleapiclient.errors import HttpError
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+class GoogleCloudComputeAPI:
+    def __init__(self, service, project_id: str):
+        self.service = service
+        self.project_id = project_id
+
+    def get_firewall(self, firewall: str):
+        request = self.service.firewalls().get(
+            project=self.project_id,
+            firewall=firewall
+        )
+        return self.execute(request)
+
+    def list_firewalls(self, filter: None, max_results: None, order_by: None, page_token: None):
```

Comment:
The `filter` argument shadows a built-in, can you rename it? Even something like `filter_` is fine to avoid collision.